_posts/doc/2017-04-26-kura-azure.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: page
+title:  "Azure IoT Hub&trade; platform"
+categories: [doc]
+---
+
+[Get Azure IoT Hub information](#azure)
+
+[Configuring a Kura Cloud Stack for Azure IoT Hub&trade;](#configuration)
+
+[Connect/Disconnect](#connection)
+
+[Kura Application Connecting to Azure IoT Hub](#kura-app)
+
+Eclipse Kura can connect to the Azure IoT Hub using the MQTT protocol. When doing so, Kura applications can send device-to-cloud messages. More information on the Azure IoT Hub and its support for the MQTT protocol can be found <a href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support" about="_blank">here</a>. This document outlines how to configure and connect a Kura application to the Azure IoT Hub.
```

Comment:
Can you please change ``Eclipse Kura`` to ``As of release 3.0 Eclipse Kura``? (no need to indicate the micro segment of the version)